tacs/problems/modal.py

```diff
@@ -525,3 +525,6 @@ def writeSolution(self, outputDir=None, baseName=None, number=None, indices=None
                 modeName = baseName + "_%3.3d" % index
                 fileName = os.path.join(outputDir, modeName) + ".f5"
                 self.outputViewer.writeToFile(fileName)
+
+    def write_sensitivity_file(self, evalFuncs, sens_file):
+        return super().write_sensitivity_file(evalFuncs, sens_file)
```

Comment:
This seems unnecessary, the method is already defined in `baseProblem`. Please remove

Reply:
Ok I will take it out, VScode auto-added those for me.